Python/cp/max_sum_k_consecutive_elements.py

```diff
@@ -0,0 +1,21 @@
+def maxSumofKelements(arr, n, k):
+    if(k > n):
+        return -1;                      #k must be smaller than n
+    ans = 0;
+    for i in range(0, k):              #first we find sum of first k elements
+        ans += int(arr[i])
+    curr_sum = ans                     #curr_sum is the current sum
+    for i in range(k, n):
+        curr_sum += int(arr[i]) - int(arr[i - k])   
+        ans = max(ans, curr_sum)
+    return ans
+    
+
```

Comment:
Add input and output messages

Reply:
Done

Reply:
perfect now add a sample test case in comment

Reply:
done